arbitrum-docs/how-arbitrum-works/timeboost/gentle-introduction.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: 'A gentle Introduction: Timeboost'
+sidebar_label: 'A gentle introduction'
+description: 'Learn how Timeboost works and how it can benefit your Arbitrum-based project.'
+author: leeederek
+sme: leeederek
+user_story: As a current or prospective Arbitrum user, I want to understand how Timeboost works and how to use it.
+content_type: gentle-introduction
+---
+
+import ImageWithCaption from '@site/src/components/ImageCaptions/';
+
+This introduction will walk you through <a data-quicklook-from='timeboost'>Arbitrum Timeboost</a>: a novel <a data-quicklook-from='transaction-ordering-policy'>transaction ordering policy</a> for Arbitrum chains that allows chain owners to capture the Maximal Extractable Value (MEV) on their chain and reduce spam, all while preserving fast block times and protecting users from harmful types of MEV, such as sandwich attacks and front-running.
+
+Timeboost is the culmination of over a year of research and development by the team at Offchain Labs and will soon be available for any Arbitrum chain, including Arbitrum One and Arbitrum Nova (should the Arbitrum DAO choose to adopt it). Like all features and upgrades to the Arbitrum technology stack, Timeboost will be rolled out on Arbitrum Sepolia first for testing, and chain owners will have the option of adopting Timeboost at any time or customizing Timeboost in any way they choose.
+
+### In a nutshell
+
+- The current <a data-quicklook-from='first-come-first-serve-fcfs'>"First-Come, First-Serve (FCFS)"</a> ordering policy has many benefits, including a great UX and protection from harmful types of MEV. However, nearly all MEV on the chain is extracted by searchers who invest wastefully in hardware and spamming to win latency races (which negatively strains the network and leads to congestion). Timeboost is a new transaction ordering policy that preserves many of the great benefits of FCFS while unlocking a path for chain owners to capture some of the available MEV on their network and introducing an auction to reduce latency, racing, and, ultimately, spam.
+- Timeboost introduces a few new components to an Arbitrum chain’s infrastructure: a sealed-bid second-price auction and a new <a data-quicklook-from='express-lane'>"express lane"</a> at an Arbitrum chain’s sequencer. Valid transactions submitted to the express lane will be sequenced immediately with no delay, while all other transactions submitted to the chain’s sequencer will experience a nominal delay (default: 200ms). The auction winner is granted the sole right to control the express lane for pre-defined, temporary intervals.
```

Comment:
Is it worth adding here that block production time is maintained at 250ms? 
It is mentioned in the sections below, but I think it's low lift to add it also here and it might prevent potential misunderstandings.